### PR TITLE
Improve stacktrace to fix tech debts

### PR DESCRIPTION
**Pull request by divyansh42**

### CODE REVIEW

File: pkg/docker/docker.go [hunk 1]
```diff
@@ -995,7 +995,14 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 		if signal == syscall.SIGQUIT {
 			buf := make([]byte, 1<<16)
 			runtime.Stack(buf, true)
-			fmt.Printf("%s", buf)
+            f, err := os.Create("/var/log/s2i_docker_stack_trace.log")
+            if err != nil {
+                return
+            }
+            defer f.Close()
+            w := bufio.NewWriter(f)
```
Comment (coreydaley) [inline pkg/docker/docker.go:1003]: I think you forgot to import this package ...

Comment (divyansh42) [reply pkg/docker/docker.go:1004]: Updated.

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,7 @@
 /.project
 /.vagrant
 .vscode/
+.idea/
```
Comment (coreydaley) [inline .gitignore:6]: This should be in your global gitignore on your workstation, not here. Go ahead and remove the .vscode one also.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

**Review by coreydaley — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (divyansh42) [reply .gitignore]: Thanks @coreydaley for providing the information

**Pull request merged**